test/flows/simplified_expenses_checker_flow_test.rb

```diff
@@ -0,0 +1,584 @@
+
+      assert_rendered_outcome text: "You would probably be better off working out your expenses based on the actual costs"
+    end
+
+    should "not render vehicle cost potential claim text, in the simplified expenses section, if the user has claimed capital expenses" do
+      add_responses capital_allowances?: "capital_allowance"
+
+      assert_no_match "for your car or van", @test_flow.outcome_text
+    end
+
+    should "not render vehicle cost potential claim text, in the simplified expenses section, if the simple vehicle cost for van/car is 0" do
+      add_responses capital_allowances?: "no",
+                    car_status_before_usage?: "new",
+                    how_much_expect_to_claim?: "10",
+                    is_vehicle_green?: "low",
+                    price_of_vehicle?: "50",
+                    vehicle_business_use_time?: "50",
+                    drive_business_miles_car_van?: "0",
+                    hours_work_home?: "100",
+                    current_claim_amount_home?: "1000"
+
+      assert_no_match "for your car or van", @test_flow.outcome_text
+    end
+
+    should "render text in the simplified expenses section if the simple vehicle costs for motorcycles is above 0" do
+      add_responses vehicle_expense?: "motorcycle",
+                    capital_allowances?: "no",
+                    how_much_expect_to_claim?: "10",
+                    price_of_vehicle?: "50",
+                    vehicle_business_use_time?: "50",
+                    drive_business_miles_motorcycle?: "10",
+                    hours_work_home?: "100",
+                    current_claim_amount_home?: "1000"
+
+      assert_rendered_outcome text: "to claim for motorcycles"
+    end
+
+    should "render text for working from home, in the simplified expenses section, if the hours worked at home is above 25" do
+      assert_rendered_outcome text: "to claim for working from home"
+    end
+
+    should "render text for living on business premises, in the simplified expenses section, if the number of people living on the premises is grater than 0" do
+      add_responses home_or_business_premises_expense?: "live_on_business_premises",
+                    deduct_from_premises?: "40000",
+                    people_live_on_premises?: "2"
+
+      assert_rendered_outcome text: "to deduct from your total business costs - you then claim the balance"
+    end
+
+    should "render text if the user has claimed running costs for the vehicle" do
+      add_responses home_or_business_premises_expense?: "no_expense",
+                    buying_new_vehicle?: "new",
+                    how_much_expect_to_claim?: "50000",
+                    is_vehicle_green?: "low",
+                    price_of_vehicle?: "50",
+                    vehicle_business_use_time?: "50",
+                    drive_business_miles_car_van?: "1000"
+
+      assert_rendered_outcome text: "for the running costs of your car, van or motorcycle"
+    end
+
```

Comment:
I think we're missing tests for lines 82 - 91 of the outcome. Testing for `of capital allowances for your car, van or motorcycle` and `for working from home`

Reply:
Ahhh I think I omitted tests here because essentially that logic had already been tested for lines 35 - 49 in the outcome. I can flesh out the commit message to make that clearer if you agree with my assessment.